components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -101,9 +104,6 @@ export const WorkspaceEntry: FunctionComponent<Props> = ({ info, shortVersion })
 };
 
 export function getProjectPath(ws: Workspace) {
-    if (CommitContext.is(ws.context)) {
-        return `${ws.context.repository.host}/${ws.context.repository.owner}/${ws.context.repository.name}`;
-    } else {
-        return undefined;
-    }
+    // TODO: respond context (source) field for Workspace?
```

Comment:
Can we get it from `ws.status.git`? do we really need it or context URL works good as well?

Reply:
`contextURL` maybe refer to prebuild? We have some other contexts.

Not sure about `ws.status.git`, since it can be changed by user behavior which is not align to old implement but seems we want this feature in future?

Reply:
Yeah, we don't want to show outdated stuff. It is a bug. 

> contextURL maybe refer to prebuild? We have some other contexts.

Context URL in Gitpod V1 is always normalized. All prefixes should be stripped already. Maybe we try with it. cc @geropl wdyt here?

The idea was that we have ws git status, and then there is `ContextSerivce.resolveContext(ws.git.status)` which knows about SCMs and can resolve the rest. We don't want to implement as part of the migration though. So maybe adding repository to git status is alright for now. We can deprecated this property later.